lib/lib.rs

```diff
@@ -44,13 +44,17 @@ pub mod rpcs;
 pub mod types;
 mod validation;
 mod verbosity;
+mod verification;
+mod verification_types;
 
+use std::{env::current_dir, path::Path};
+#[cfg(feature = "std-fs-io")]
 use std::{
     fs,
     io::{Cursor, Read, Write},
-    path::Path,
 };
 
+#[cfg(feature = "std-fs-io")]
```

Comment:
I would also expect a change on the method too if you add the flag here

```
#[cfg(feature = "std-fs-io")]
pub(crate) fn json_pretty_print<T: ?Sized + Serialize>(
```

Can you please check ? either remove those flags or surround the method using this Serialize too.